src/server/api/modules/recipes/service/schemas.ts

```diff
@@ -18,7 +23,9 @@ export const RecipeImageUrlSchema = z
   })
 
 export const JsonLdRecipeSchema = z.object({
-  "@type": z.union([z.string(), z.tuple([z.string()]).transform((a) => a[0])]),
+  "@type": z.union([z.string(), z.string().array().nonempty()]).transform((o) => {
```

Comment:
```suggestion
  "@type": z.union([z.string(), z.string().array()]).transform((o) => {
```
again with the nonempty